### DISCUSSION TITLE
Ensure continuity of gam from start of series to end of series

### DISCUSSION THREAD

**Discussion by markbneal** [gavinsimpson/gratia#276] [Ideas]
For seasonal data, there should be no discontinuity from start to end value. I may not have the terms exactly correct, but I see cyclic splines mentioned here as resolving this. Not being my expertise, is it in scope for gratis and straightforward to show an example in documentation as per one of your old blog posts?

https://fromthebottomoftheheap.net/2014/05/09/modelling-seasonal-data-with-gam/

**Comment by gavinsimpson** [comment]
I'm not sure I understand what you are asking, Mark? gratia doesn't do any model fitting itself, so the model fitted in the blog post you mention could be plotted using gratia with 

```r
draw(m)
``` 

instead of 

```r
layout(matrix(1:2, ncol = 2))
plot(m$gam, scale = 0)
layout(1)
```

From the point of view of gratia, once the smooth has been defined using mgcv's smooth function, `s()` etc, or estimated via a modelling function (usually one of mgcv's, but others are supported with more planned), it can be plotted with `draw()`.

I doubt I'll add something on this specifically to the documentation of the package, but I might do another blog post on these in the near future. Anything I blog about now will almost certainly feature gratia prominently, as the primary purpose for writing it was to facilitate my own use of GAMs in my day job.

I'll convert this to a discussion and you can add any follow up there, perhaps toclarify your ask if I have misunderstood it.
